local-tests/setup/tinny-environment.ts

```diff
@@ -0,0 +1,366 @@
+import { LIT_TESTNET, ProcessEnvs, TinnyEnvConfig } from './tinny-config';
+import { LitNodeClient } from '@lit-protocol/lit-node-client';
+import { LitContracts } from '@lit-protocol/contracts-sdk';
+import { AuthSig, LitContractContext } from '@lit-protocol/types';
+import { TinnyPerson } from './tinny-person';
+import networkContext from './networkContext.json';
+import { ethers } from 'ethers';
+import { LIT_ENDPOINT, LIT_ENDPOINT_VERSION } from '@lit-protocol/constants';
+
+export class TinnyEnvironment {
+  public network: LIT_TESTNET;
+
+  /**
+   * Environment variables used in the process.
+   */
+  public processEnvs: ProcessEnvs = {
+    MAX_ATTEMPTS: parseInt(process.env['MAX_ATTEMPTS']) || 3,
+    NETWORK: (process.env['NETWORK'] as LIT_TESTNET) || LIT_TESTNET.LOCALCHAIN,
+    DEBUG: Boolean(process.env['DEBUG']) || false,
+    REQUEST_PER_KILOSECOND:
+      parseInt(process.env['REQUEST_PER_KILOSECOND']) || 200,
+    LIT_RPC_URL: process.env['LIT_RPC_URL'] || 'http://127.0.0.1:8545',
+    WAIT_FOR_KEY_INTERVAL:
+      parseInt(process.env['WAIT_FOR_KEY_INTERVAL']) || 3000,
+    BOOTSTRAP_URLS: process.env['BOOTSTRAP_URLS']?.split(',') || [
+      'http://127.0.0.1:7470',
+      'http://127.0.0.1:7471',
+      'http://127.0.0.1:7472',
+    ],
+    LIT_OFFICIAL_RPC:
+      process.env['LIT_OFFICIAL_RPC'] ||
+      'https://chain-rpc.litprotocol.com/http',
+    TIME_TO_RELEASE_KEY: parseInt(process.env['TIME_TO_RELEASE_KEY']) || 10000,
+    RUN_IN_BAND: Boolean(process.env['RUN_IN_BAND']) || false,
+    RUN_IN_BAND_INTERVAL: parseInt(process.env['RUN_IN_BAND_INTERVAL']) || 5000,
+
+    // Available Accounts
+    // ==================
+
+    // (0) "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266" (10000.000000000000000000 ETH)
+    // (1) "0x70997970C51812dc3A010C7d01b50e0d17dc79C8" (10000.000000000000000000 ETH)
+    // (2) "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC" (10000.000000000000000000 ETH)
+    // (3) "0x90F79bf6EB2c4f870365E785982E1f101E93b906" (10000.000000000000000000 ETH)
+    // (4) "0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65" (10000.000000000000000000 ETH)
+    // (5) "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc" (10000.000000000000000000 ETH)
+    // (6) "0x976EA74026E726554dB657fA54763abd0C3a0aa9" (10000.000000000000000000 ETH)
+    // (7) "0x14dC79964da2C08b23698B3D3cc7Ca32193d9955" (10000.000000000000000000 ETH)
+    // (8) "0x23618e81E3f5cdF7f54C3d65f7FBc0aBf5B21E8f" (10000.000000000000000000 ETH)
+    // (9) "0xa0Ee7A142d267C1f36714E4a8F75612F20a79720" (10000.000000000000000000 ETH)
```

Comment:
1. Where do these come from?
2. Can we define more?
3. If we can define more, can we just make a very large pool of e.g. 10000 of them, and remove the 'reserve private keys' code entirely?

Reply:
these are default keys from a local chain both `anvil` and `hardhat` use these as defaulted test addresses. I think we should be using this in `local dev` since they are known to be test keys.